apps/sample-configs/osvr_server_config.moverio_highfov.json

```diff
@@ -0,0 +1,98 @@
+{
+  "description": "This configuration supports video (so-called 'positional') and IMU fusion tracking, in addition to orientation-only tracking, with the OSVR HDK 2, for units that were manufactured as HDK2 (NOT UPGRADED FROM HDK 1.x!). It is configured for RenderManager applications in direct mode on HDK2 optics.",
+  "display": {
+  "meta": {
+    "schemaVersion": 1
+  },
+  "hmd": {
+    "device": {
+      "vendor": "Epson",
+      "model": "Moverio BT-300 series",
+      "num_displays": 1,
+      "Version": "1.0",
+      "Note": "This is not the native FOV of the device but a higher FOV for 'magic window' style display."
+    },
+    "field_of_view": {
+      "monocular_horizontal": 90,
+      "monocular_vertical": 50.98299243,
+      "overlap_percent": 100,
+      "pitch_tilt": 0
+    },
+    "resolutions": [
+      {
+        "width": 1280,
+        "height": 720,
+        "video_inputs": 1,
+        "display_mode": "horz_side_by_side",
+        "swap_eyes": 0
```

Comment:
this line is default and can be omitted. Same comments from previous file apply to this as well.